jenkins2/init.groovy

```diff
@@ -0,0 +1,12 @@
+import hudson.model.*;
+import jenkins.model.*;
+
+
+Thread.start {
+      sleep 10000
+      println "--> setting agent port for jnlp"
+      def env = System.getenv()
+      int port = env['JENKINS_SLAVE_AGENT_PORT'].toInteger()
+      Jenkins.instance.setSlaveAgentPort(port)
+      println "--> setting agent port for jnlp... done"
```

Comment:
we should also be setting the tz to Europe/London so job timestamps are not in utc

https://wiki.jenkins-ci.org/display/JENKINS/Change+time+zone